java/src/main/java/ai/rapids/cudf/StripType.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ *  Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package ai.rapids.cudf;
+
+public enum StripType {
```

Comment:
Can we make this private?  There are no public APIs that use it, so it is an implementation detail at this point.

Reply:
Thanks for pointing it out @revans2 . I have moved this to ColumnVector class and made it private.